tests/test_collectors.py

```diff
@@ -563,24 +563,39 @@ def define_constraints(constraint_factory: ConstraintFactory):
     assert score_manager.explain(problem).score == SimpleScore.of(4)
 
 
-def test_flatten_last():
+def test_load_balance():
```

Comment:
I am certain that the test is wrong; it is not finished. Unfortunately, I can not tell what's wrong with the test, because the only message I get is:

    java.lang.java.lang.IllegalStateException: java.lang.IllegalStateException: Consequence of a constraint (org.jpyinterpreter.user.tests.test_collectors/Balanced value) threw an exception processing a tuple ({ai.timefold.solver.core.impl.score.stream.collector.LoadBalanceImpl@5c945ee7}).

See that the exception cause is missing. I'm assuming this is something that needs to be fixed in the interpreter?

Please help me fix this, the PR needs to be merged ASAP.